### PR TITLE
Update Toast ui

### LINKED ISSUE #138: Add prop to change default icon in Toast ui

### PR DESCRIPTION
**Pull request by ckrac**
- add prop to change default icon in Toast ui
- update iconic package to latest version

closes #138

### CODE REVIEW

**Comment by ckrac** [comment]
- build commit is to use for areo-web development until merged (will remove once reviewed)

File: src/ui/src/components/Toast/Toast.stories.tsx [hunk 1]
```diff
@@ -2,10 +2,12 @@ import React, { FC } from 'react'
 import { storiesOf } from '@storybook/react'
 import { text, boolean } from '@storybook/addon-knobs'
 import { Toast, EToastStatus, EColor } from '@src/index'
+import { ToastBrio } from '@areo/iconic'
 
 const props = () => ({
   message: text('message', 'Message'),
   showCloseBtn: boolean('showCloseBtn', true),
+  CustomIcon: boolean('show customIcon', false) ? ToastBrio : undefined,
```
Comment (asucarlos) [inline src/ui/src/components/Toast/Toast.stories.tsx:10]: Correct me if I'm wrong, but I can see in the design that the icon and color changes together like any other variant.
Why not add a BRIO variant to ToastStatus and ToastStatusIconAndColor constants?

**Review by asucarlos — CHANGES_REQUESTED** [review]

File: src/ui/src/components/Toast/Toast.stories.tsx (hunk 1 above)
Comment (ckrac) [reply src/ui/src/components/Toast/Toast.stories.tsx:10]: > Correct me if I'm wrong, but I can see in the design that the icon and color changes together like any other variant.
> Why not add a BRIO variant to ToastStatus and ToastStatusIconAndColor constants?

The user of this library will have to add a new prop and update the icon package every time a new icon is added to use with this ui. This way we can eliminate that and color changes can be added to the Icon element passed if they want to change the color as well.

Comment (asucarlos) [reply src/ui/src/components/Toast/Toast.stories.tsx:10]: ahh, makes sense :) I was confusing stories and Toast component.

**Review by asucarlos — APPROVED** [review]

**Pull request merged**